devsetup/edpm/edpm-play.yaml

```diff
@@ -107,14 +108,20 @@ spec:
                   ctlplane_dns_nameservers:
                   - 192.168.122.1
                   dns_search_domains: []
-                  tripleo_nova_compute_DEFAULT_transport_url: rabbit://guest:ar3TXOAYMaznW29fRLj2ILXKs@standalone.ctlplane.localdomain:5672/?ssl=0
-                  tripleo_nova_compute_cache_memcache_servers: standalone.ctlplane.localdomain:11211
+                  tripleo_nova_compute_DEFAULT_transport_url:
+                    "rabbit://guest:ar3TXOAYMaznW29fRLj2ILXKs@standalone.ctlplane.\
+                    localdomain:5672/?ssl=0"
```

Comment:
The format of strings on multiple lines should be kept the same if possible.

Reply:
I'm also in favor of keeping line on one line.. But since yamllint line length is set for 100 so we should exceed more than it.

Sometimes breaking is good option. Added with "" and \ is to make url consistent. 

I've tried to use other suggested approach but results are not promising
```
---
- hosts: localhost
  gather_facts: false
  vars:
    url1: >-
       "https://github.com/operator-framework/operator-registry/release/download/\
        v1.26.3/linux-amd64-opm"
    url2: >-
        https://github.com/operator-framework/operator-registry/release/download/
        v1.26.3/linux-amd64-opm
    url3:
        "https://github.com/operator-framework/operator-registry/release/download/\
        v1.26.3/linux-amd64-opm"
    url4: >- 
      https://github.com/operator-framework/operator-registry/release/download/
      v1.26.3/linux-amd64-opm
  tasks:
    - name: "debug"
      debug:
        var: "{{ item }}"
      with_items:
        - url1
        - url2
        - url3
        - url4
```
And it outputs
```
TASK [debug] *******************************************************************************************
ok: [localhost] => (item=url1) => {
    "ansible_loop_var": "item",
    "item": "url1",
    "url1": "\"https://github.com/operator-framework/operator-registry/release/download/\\\n v1.26.3/linux-amd64-opm\""
}
ok: [localhost] => (item=url2) => {
    "ansible_loop_var": "item",
    "item": "url2",
    "url2": "https://github.com/operator-framework/operator-registry/release/download/ v1.26.3/linux-amd64-opm"
}
ok: [localhost] => (item=url3) => {
    "ansible_loop_var": "item",
    "item": "url3",
    "url3": "https://github.com/operator-framework/operator-registry/release/download/v1.26.3/linux-amd64-opm"
}
ok: [localhost] => (item=url4) => {
    "ansible_loop_var": "item",
    "item": "url4",
    "url4": "https://github.com/operator-framework/operator-registry/release/download/ v1.26.3/linux-amd64-opm"
}
```
So keeping it in the string for now (Resolving below discussions related to strings).. waiting for more suggestions.